.gitignore

```diff
@@ -160,3 +160,15 @@ logviewer/pnpm-lock.yaml
 logviewer/webpack.config.js
 logviewer/webpack.generated.js
 .DS_Store
+
+# VisualStudioCode
+.vscode/*
+!.vscode/settings.json
+!.vscode/extensions.json
+!.vscode/*.code-snippets
+
+# Local History for Visual Studio Code
+.history/
+
+# Built Visual Studio Code Extensions
+*.vsix
```

Comment:
Seems your PR contains changes from the other PR. Either remove them from this branch or we have to block this PR until that other PR is merged.